neurodamus/io/sonata_config.py

```diff
@@ -39,6 +41,7 @@ def __init__(self, config_path):
         self._sim_conf = libsonata.SimulationConfig.from_file(config_path)
         self._entries = {}
         self._sections = {}
+        self._internal_edge_pops = set()
```

Comment:
Why is this a set if somewhere else we say that order matters?

Do we need this as an additional attribute, too? It seems to me that `circuit_conf["nrnPath"]` has the same contents and could be queried instead?

Reply:
This set is internal, to keep track which edge populations are already "exposed" as internal connectivity, so the remaining are exposed as projections. For this purpose we are better served as a set and no need to expose as a property.

Reply:
Yes, but could we not just construct this set on the fly before instantiating all these "projections"?

Reply:
We have `Circuit` and `Projection` properties which both consume edges. Hence this change needs a shared object